src/services/exposed/error-handler.js

```diff
@@ -32,6 +36,7 @@ function errorHandler({ logger } = {}) {
           status: error.status || 500,
           detail: error.message,
           name: error.name,
```

Comment:
Here you don't follow the same principle than on the `agent-nodejs` by using the constructor's name?

Reply:
Well, I hadn't looked at all errors. It seems that all used `this.name = this.constructor.name;` syntax in constructor. So let's change it.